v2/middlewares/tracing/tracing.go

```diff
@@ -33,6 +35,8 @@ func (t tracingMw) Process(ctx context.Context, item *kafka.Message, next func(c
 	return err
 }
 
-func NewTracingMiddleware() middleware.Middleware[*kafka.Message, error] {
-	return tracingMw{}
+func NewTracingMiddleware(tracerProvider *trace.TracerProvider) middleware.Middleware[*kafka.Message, error] {
+	return &tracingMw{
```

Comment:
```suggestion
	return &tracingMiddleware{
```

Reply:
Yes I think this change is more readable